tests/fixtures/generic.py

```diff
@@ -1,3 +1,4 @@
+"""Fixtures for general use."""
 from sleap_io import (
     Video,
     Point,
```

Comment:
We should rename this file and split-up the fixtures here into more descriptive/easy-to-find files. Also, I did not see these fixtures being used anywhere in the repo.